src/img/ico-layer-delete.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="layer-delete" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
+	 y="0px" viewBox="0 0 612 792" style="enable-background:new 0 0 612 792;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:none;stroke:#000000;stroke-width:80;stroke-miterlimit:10;}
+</style>
+<g transform="translate(1 1)">
+	<path class="st0" d="M408.6,291.4L201.4,498.6 M201.3,291.3l207.3,207.3"/>
+</g>
+</svg>

```

Comment:
Got it from @designedbyscience as a temp icon, since we are moving toward UI redesign.